template_feed/Microsoft.DotNet.Common.ItemTemplates/content/EditorConfig/Dotnet/.editorconfig

```diff
@@ -1,364 +1,132 @@
+###############################
+# Core EditorConfig Options   #
+###############################
 root = true
-
 # All files
 [*]
 indent_style = space
 
-# Xml files
-[*.xml]
+# XML project files
+[*.{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}]
```

Comment:
This lacks support for F# project files, fsproj.

Reply:
Good catch but do you want to change it here when the template in VS2022 has the same problem?

Reply:
That's more a note that we should resolve it in both places. Fixing it upstream in VS2022 first, then flowing down to here works for me. Where would that be reported?

Reply:
@baronfel file on roslyn please